src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Figure defaultFigure = new Circle(Color.WHITE, 10);
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final double maxValue = 10;
+    private final int amountOfFigures = 5;
```

Comment:
The maxValue and amountOfFigures should be constants according to the checklist. They should be declared as 'public static final' and their names should be in uppercase (e.g., MAX_VALUE, AMOUNT_OF_FIGURES).

Reply:
The maxValue and amountOfFigures should be constants according to the checklist. They should be declared as 'public static final' and their names should be in uppercase (e.g., MAX_VALUE, AMOUNT_OF_FIGURES).